modules/upcloud.py

```diff
@@ -203,29 +206,42 @@ def find_server(self, uuid, hostname):
 
         return None
 
-
-    def create_server(self, module):
-        """Create a server from module.params. Filters out unwanted attributes."""
+    def create_server(self, module_params):
```

Comment:
Only params are needed to pass and not the whole module, plus it makes it easier to make tests since it is not needed to mock the whole module but only the params in a form of a dict.